docs/buildah-build.1.md

```diff
@@ -586,6 +586,10 @@ include:
   "sigpending": maximum number of pending signals (ulimit -i)
   "stack": maximum stack size (ulimit -s)
 
+**--unsetenv** *env*
+
+Unset environment variables from the final image.
+
 **--userns** *how*
```

Comment:
Isn’t $PATH the only use case?  Thus thereis no need for a parameter to the --unsetenv command line option.

Reply:
I still think the PATH variable is the only use case and therefore this switch shall be called `--unsetpathevn` without parameters.